### PR TITLE
Prepare serve image

### PR DESCRIPTION
**Pull request by Lev1ty**

### CODE REVIEW

File: gunzip.py
Comment (Zack-Ren) [inline gunzip.py:1]: whats the purpose of this and `lzma.py`?

Comment (Lev1ty) [reply gunzip.py:1]: to extract *.gz and *.xz files without pulling in the gunzip and xz binaries

Comment (Zack-Ren) [reply gunzip.py:1]: would it be cleaner to put these in a util package?

Comment (Lev1ty) [reply gunzip.py:1]: do you want to do that? would be a good starter task to learn bazel

**Review by Zack-Ren — APPROVED** [review]

**Pull request merged**